desktop/src/apps/ProjectsApp/board/BoardColumn.module.css

```
@@ -4,4 +4,4 @@
 .count { margin-left: auto; font: 600 11px -apple-system; background: rgba(255,255,255,.07); padding: 2px 8px; border-radius: 8px; color: #d2d2d7; }
 .body { display: flex; flex-direction: column; gap: 10px; }
 .showMore { background: transparent; border: 1px dashed var(--board-hair); color: #86868b; padding: 6px; border-radius: 8px; cursor: pointer; font: 11px -apple-system; }
-.ready, .claimed, .closed {} /* status modifiers reserved for future glow */
+.ready, .claimed, .closed, .quarantined {} /* status modifiers reserved for future glow */
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)